src/components/profile/Profile.jsx

```diff
@@ -23,6 +24,14 @@ const Profile = () => {
             ))}
           </ul>
         )}
+        <h3>My Rockets</h3>
```

Comment:
A good idea would be if you changed this heading instead of "My Rockets" could be "My Missions" because there is already one "My Rockets" section.